src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import commentsFromServer from '../../api/comments';
+
+import { Post } from '../../Types/post';
+import { Comment } from '../../Types/comment';
+
+import { PostInfo } from '../PostInfo';
+
+function getCommit(postId: number): Comment[] {
+  return commentsFromServer.filter(comment => comment.postId === postId);
+}
+
+type Props = {
+  posts: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="PostList">
```

Comment:
Add a CSS import statement for the 'PostList' component to import the appropriate styles.